client/src/components/select-formula-view/ViewFormulaAccordion.js

```diff
@@ -0,0 +1,50 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import Accordion from '@mui/material/Accordion';
+import AccordionSummary from '@mui/material/AccordionSummary';
+import AccordionDetails from '@mui/material/AccordionDetails';
+import Typography from '@mui/material/Typography';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import Container from '@mui/material/Container';
+import styled from 'styled-components';
+
+const HoverExpandMoreIcon = styled(ExpandMoreIcon)`
+  &:hover {
+    background: #f4f4f4;
+  }
+`;
+
+const ViewFormulaAccordion = ({ codeSnippet }) => {
+
+  return (
+    <Container>
+      <Accordion sx ={{ boxShadow: 'none' }}>
+        <AccordionSummary
+          expandIcon={<HoverExpandMoreIcon />}
+          aria-controls="panel1a-content"
+          id="panel1a-header"
+          sx ={{ pl: 0 }}
+        >
+          <Typography>Preview of the formula</Typography>
+        </AccordionSummary>
+        <AccordionDetails sx={{ bgcolor: 'primary.light', overflow: 'scroll' }}>
```

Comment:
To align the code correctly to the center these changes could be added:
```
        <AccordionDetails sx={{ bgcolor: 'primary.light', overflowX: 'scroll', p: '10px' }}>
          <Container disableGutters>
```
Also some padding to the left of the code when scrolling would be nice but I couldn't figure this out yet since I couldn't get scroll-padding to work. So, if this a too difficult task then it can be omitted.